### ISSUE TITLE
Minimize Brightness on Non-AMOLED Devices When Display is Turned Off

### ISSUE THREAD

**Issue by Irfan234-afif** [Genymobile/scrcpy#4597]
Hi scrcpy team,

I hope this message finds you well. I am a user who greatly appreciates the functionality and convenience that scrcpy provides. I've noticed a small issue when using scrcpy with non-AMOLED devices and the screen is turned off using the -S parameter.

The problem arises when the screen displays a black color, causing non-AMOLED screens to show a bluish tint rather than turning the pixels off completely. To address this, I would like to propose a feature that minimizes the brightness to the lowest possible level when the screen is turned off.

Please note that on some devices, setting the brightness to 0 through adb shell may not be supported. Therefore, I suggest setting the brightness to 1 as an alternative to achieve the desired effect. This would not only conserve energy but also contribute to the longevity of non-AMOLED screens.

I believe implementing this feature, with the suggested adjustment, would enhance the user experience, especially for those with non-AMOLED devices.

Thank you for your time and consideration.

Best regards,
Irfan

**Comment by rom1v** [comment]
Even on (some) non-amoled devices, turning off the screen actually turns the pixels off completely (it depends on the device/system).

**Comment by Irfan234-afif** [comment]
> Even on (some) non-amoled devices, turning off the screen actually turns the pixels off completely (it depends on the device/system).

Yes, but on some non-AMOLED devices, when the device turns off, it doesn't completely turn off the screen pixels. I tried minimizing the brightness before opening scrcpy, and my screen looks better, appearing as if it's completely turned off (even though it might not look like it).

**Comment by Daodanfd5** [comment]
maybe display a completely black float window and don't capture this window?
it can truly disable all pixels.

**Comment by rom1v** [comment]
> maybe display a completely black float window

Not possible technically.
